src/data/energy.ts

```diff
@@ -94,9 +101,16 @@ export interface SolarSourceTypeEnergyPreference {
   config_entry_solar_forecast: string[] | null;
 }
 
+export interface BatterySourceTypeEnergyPreference {
+  type: "battery";
+  stat_energy_from: string;
+  stat_energy_to: string;
+}
```

Comment:
Should we add `entity_battery_charge_pct?: string`

Reply:
Dunno, dashboard only updates once an hour, we only show daily info now, this would be current info?

Reply:
good point. for future us.